SPECS/libstoragemgmt/libstoragemgmt.spec

```diff
@@ -0,0 +1,911 @@
+Summary:        Files for LSI MegaRAID support for %{name}
+Requires:       python3-%{name} = %{version}
+Requires(post): python3-%{name} = %{version}
+Requires(postun): python3-%{name} = %{version}
+BuildArch:      noarch
+
+%description    megaraid-plugin
+The %{name}-megaraid-plugin package contains the plugin for LSI
+MegaRAID storage management via storcli.
+
+%package        hpsa-plugin
+Summary:        Files for HP SmartArray support for %{name}
+Requires:       python3-%{name} = %{version}
+Requires(post): python3-%{name} = %{version}
+Requires(postun): python3-%{name} = %{version}
+BuildArch:      noarch
+
+%description    hpsa-plugin
+The %{name}-hpsa-plugin package contains the plugin for HP
+SmartArray storage management via hpssacli.
+
+%package        arcconf-plugin
+Summary:        Files for Microsemi Adaptec and Smart Family support for %{name}
+Requires:       python3-%{name} = %{version}
+Requires(post): python3-%{name} = %{version}
+Requires(postun): python3-%{name} = %{version}
+BuildArch:      noarch
+
+%description    arcconf-plugin
+The %{name}-arcconf-plugin package contains the plugin for Microsemi
+Adaptec RAID and Smart Family Controller storage management.
+
+%package        nfs-plugin
+Summary:        Files for NFS local filesystem support for %{name}
+Requires:       python3-%{name} = %{version}
+Requires:       %{name}-nfs-plugin-clibs = %{version}
+Requires:       nfs-utils
+Requires(post): python3-%{name} = %{version}
+Requires(postun): python3-%{name} = %{version}
+BuildArch:      noarch
+
+%description    nfs-plugin
+The nfs-plugin package contains plug-in for local NFS exports support.
+
+%package        nfs-plugin-clibs
+Summary:        Python C extension module for %{name} NFS plugin
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description    nfs-plugin-clibs
+The %{name}-nfs-plugin-clibs package contains python C
+extension for %{name} NFS plugin.
+
+
+%package        local-plugin
+Summary:        Files for local pseudo plugin of %{name}
+Requires:       python3-%{name} = %{version}
+Requires(post): python3-%{name} = %{version}
+Requires(postun): python3-%{name} = %{version}
+BuildArch:      noarch
+
+%description    local-plugin
+The %{name}-local-plugin is a plugin that provides auto
+plugin selection for locally managed storage.
+
+%prep
+%autosetup -p1
+
+%build
+./autogen.sh
+%configure --with-python3 --disable-static
+%make_build
+
+%install
+rm -rf %{buildroot}
+
+%make_install
+
+find %{buildroot} -name '*.la' -exec rm -f {} ';'
+
+#Files for udev handling
+mkdir -p %{buildroot}/%{_udevrulesdir}
+install -m 644 tools/udev/90-scsi-ua.rules \
+    %{buildroot}/%{_udevrulesdir}/90-scsi-ua.rules
+install -m 755 tools/udev/scan-scsi-target \
+    %{buildroot}/%{_udevrulesdir}/../scan-scsi-target
+
+mkdir -p %{buildroot}/%{_datadir}/bash-completion/completions/
+mv %{buildroot}/etc/bash_completion.d/lsmcli %{buildroot}/%{_datadir}/bash-completion/completions/
+
+%if 0%{with test}
```

Comment:
We have our own way of running the tests or not during build through the `RUN_CHECK` argument. We don't need the `%bcond_with     test` line and this `if` statement right here and they should be removed. Otherwise we will end up never running the tests, regardless of the `RUN_CHECK` argument's value.

Reply:
Done.